clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -156,6 +156,8 @@ def err_drv_unsupported_rtlib_for_platform : Error<
   "unsupported runtime library '%0' for platform '%1'">;
 def err_drv_invalid_unwindlib_name : Error<
   "invalid unwind library name in argument '%0'">;
+def err_drv_unsupported_unwind_for_platform : Error<
+  "unwind library '%0' is not supported by this platform">;
```

Comment:
perhaps the message could be made more similar to `err_drv_unsupported_rtlib_for_platform`

Reply:
I deliberately changed it because I didn't know how to describe this platform in the `%1`.

But yeah I can make it like that and use the triple I guess.